docs/user_manual.md

```diff
@@ -492,6 +492,17 @@ The `dataCenter` field in the top level of the spec specifies what data center t
 
 Replicating across data centers will likely mean running your cluster across multiple Kubernetes clusters, even if you are using a single-Kubernetes replication strategy within each DC. This will mean taking on the operational challenges described in the "Multi-Kubernetes Replication" section above.
 
+# Managing disruption
```

Comment:
Do you mind to add an example PDB to https://github.com/FoundationDB/fdb-kubernetes-operator/tree/master/config/samples that might be useful for people that want to use PDBs.